src/LaplaceApproximationModule.jl

```diff
@@ -206,7 +220,7 @@ end
 `dist_y_given_f` must be a scalar function from a Real to a Distribution object.
 `ys` and `f` are vectors of observations and latent function values, respectively.
 """
-function loglik_and_derivs(dist_y_given_f, ys::AbstractVector, f::AbstractVector{<:Real})
+function loglik_and_derivs(dist_y_given_f, ys, f)
```

Comment:
```suggestion
function loglik_and_derivs(dist_y_given_f, ys::AbstractVector, f::AbstractVector{<:Real})
```
let's try putting this back